docs/api.html

```diff
@@ -165,3 +165,22 @@
 
 Examples showing how to use the consumer are given in the
 <a href="http://kafka.apache.org/0100/javadoc/index.html?org/apache/kafka/clients/consumer/KafkaConsumer.html" title="Kafka 0.9.0 Javadoc">javadocs</a>.
+
+<h3><a id="streamsapi" href="#streamsapi">2.3 Streams API</a></h3>
+
+As of the 0.10.0 release we have added a new client library named <b>Kafka Streams</b> to let users implement their stream processing
+applications with data streams stored in Kafka. This client is considered alpha quality and its public APIs are likely to change in
```

Comment:
"This client is considered" -> "Kafka Streams is considered"

In general, I'd suggest to not use "client" when referring to Kafka Streams (yes, it is a client, but rather name it "Kafka Streams" to make a clearer distinction to the "normal" clients that Kafka folks are used to).